src/MapViewController.swift

```diff
@@ -17,12 +17,12 @@ import OnTheRoad
   case GeneralError, AnnotationDoesNotExist, APIKeyNotSet, RouteDoesNotExist
 }
 
-public class MapViewController: TGMapViewController, LocationManagerDelegate, TGRecognizerDelegate {
+public class MapViewController: UIViewController, LocationManagerDelegate, TGRecognizerDelegate {
 
   //Error Domains for NSError Appeasement
   public static let MapzenGeneralErrorDomain = "MapzenGeneralErrorDomain"
-
-
+  
+  var tgViewController: TGMapViewController = TGMapViewController()
```

Comment:
Is it similar to the use case here: https://github.com/mapzen/android/blob/master/core/src/main/java/com/mapzen/android/graphics/MapView.java#L106

Where one gives access to the underlying TangramMap object, but still the recommended way is to use the MapView (wrapper methods).